src/lwc/gsChecklist/gsChecklist.html

```diff
@@ -0,0 +1,55 @@
+<template>
+    <article class="slds-setup-assistant__step">
+        <div class="slds-summary-detail">
+            <button class="slds-button slds-button_icon slds-m-right_x-small slds-m-top_x-small"
+                id={arrowId}
+                onclick={onClickOpenOrClose}
+                aria-controls={expandedId}
+                aria-expanded={open}
+                title={group.title}>
+                <lightning-icon  
+                    alternative-text=""
+                    icon-name={arrowIcon}
+                    size="x-small"
+                    title={group.title}>
+                </lightning-icon>
+            </button>
+            <div class="slds-container_fluid">
+                <div class="slds-summary-detail__title">
+                    <div class="slds-setup-assistant__step-summary">
+                        <div class="slds-media">
+                            <c-gs-checklist-icon label={group.id}>
+                            </c-gs-checklist-icon>
+                            <div class="slds-media__body slds-m-top_x-small">
+                                <div class="slds-media">
+                                    <div class="slds-setup-assistant__step-summary-content slds-media__body">
+                                        <h3 class="slds-setup-assistant__step-summary-title slds-text-heading_small">
+                                            <button onclick={onClickOpenOrClose}
```

Comment:
can't we use a lighting-button here?

Reply:
Not, this button not have any SLDS buttons style in her